java/matcher_test.go

```diff
@@ -5,16 +5,19 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/quay/zlog"
+
 	"github.com/quay/claircore"
+	"github.com/quay/claircore/test"
 )
 
 func TestVulnerable(t *testing.T) {
 	matcher := &Matcher{}
 
 	testcases := []struct {
-		name   string
 		record *claircore.IndexRecord
 		vuln   *claircore.Vulnerability
+		name   string
```

Comment:
just curious why move this down? Seems like it tends to be the first thing set

Reply:
My editor complained about the layout causing extra padding, so I just let it rearrange the struct. The structs are all keyed, so it doesn't matter to the declarations.